### DISCUSSION TITLE
Codecentric's helm chart X Keycloakify

### DISCUSSION THREAD

**Discussion by noahra** [keycloakify/keycloakify#444] [Q&A]
I am using this Helm chart to deploy Keycloak: https://github.com/codecentric/helm-charts/tree/master/charts/keycloakx

How can I use a theme I've generated with Keycloakify on my Keycloak instance? Anyone that has tried that?

**Comment by garronej** [comment]
Hello @noahra,  
Sure, we're using this chart in our production platform.  

You can checkout the instalation guide of onyxia:  
https://docs.onyxia.sh/#enabling-user-authentication  

In perticular: 
<img width="822" alt="image" src="https://github.com/keycloakify/keycloakify/assets/6702424/7e5ad2e9-fbeb-40b8-8add-699157069933">
